tests/php/src/Internal/DependencyManagement/AbstractServiceProviderTest.php

```diff
@@ -84,47 +85,47 @@ function get_new_dependency_class( DependencyClass $dependency ) {
 	 */
 	public function test_add_with_auto_arguments_throws_on_non_class_passed_as_class_name() {
 		$this->expectException( ContainerException::class );
-		$this->expectExceptionMessage( "AbstractServiceProvider::add_with_auto_arguments: error when reflecting class 'foobar': Class foobar does not exist" );
+		$this->expectExceptionMessage( "You cannot add 'foobar', only classes in the Automattic\WooCommerce\ namespace are allowed." );
 
 		$this->sut->add_with_auto_arguments( 'foobar' );
 	}
 
 	/**
-	 * @testdox 'add_with_auto_arguments' should throw an exception if the passed class has a private constructor.
+	 * @testdox 'add_with_auto_arguments' should throw an exception if the passed class has a private method.
```

Comment:
Better "...has a private injection method". The same for the other tests with a similar comment.